corehq/apps/domain/forms.py

```diff
@@ -956,6 +958,19 @@ class DomainInternalForm(forms.Form, SubAreaMixin):
         required=False,
         min_value=1,
     )
+    use_custom_gsheet_limit = forms.ChoiceField(
+        label=ugettext_lazy("Set custom Google Sheet limit? Default is 20"),
```

Comment:
The 20 here is hardcoded because we need a get_default_limit function that is being introduced in a different branch. I will update this code in that branch.